program/rust/src/processor/upd_price.rs

```diff
@@ -190,6 +190,12 @@ pub fn upd_price(
                 clock.slot,
                 clock.unix_timestamp,
             );
+            let agg_diff = {
+                (clock.slot as i64)
+                    - load_checked::<PriceAccount>(price_account, cmd_args.header.version)?
+                        .prev_slot_ as i64
```

Comment:
so the `prev_slot` after `c_upd_aggregate` will remain the same as `last_slot`before calling `c_upd_aggregate`?

Reply:
yup since both `prev_slot` and `last_slot` is only updated in `c_upd_aggregate` if `ptr->agg_.status_ == PC_STATUS_TRADING`